package.json

```diff
@@ -87,6 +87,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "mongoose": "^5.9.17",
-    "node-fetch": "^2.6.0"
+    "node-fetch": "^2.6.0",
+    "react": "^16.13.1"
```

Comment:
please do not add react to the root package.json

Reply:
Done